target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -201,6 +201,11 @@ ramips_setup_interfaces()
 	wl-330n)
 		ucidef_set_interface_lan "eth0.1"
 		;;
+	mr200)
+		ucidef_add_switch "switch0" \
+			"0:lan" "1:lan" "2:lan" "3:lan" "6t@eth0"
+		ucidef_set_interface_wan "usb0"
+		;;
```

Comment:
On my mr200 this config always  work good: 
```
config interface 'loopback'
        option ifname 'lo'
        option proto 'static'
        option ipaddr '127.0.0.1'
        option netmask '255.0.0.0'

config globals 'globals'
        option ula_prefix 'fd1f:2c01:2cde::/48'

config interface 'lan'
        option type 'bridge'
        option ifname 'eth0.1'
        option proto 'static'
        option ipaddr '192.168.1.1'
        option netmask '255.255.255.0'
        option ip6assign '60'

config device 'lan_dev'
        option name 'eth0.1'
        option macaddr '3c:46:d8:aa:89:4f'

config interface 'wan'
        option ifname 'usb0'
        option proto 'dhcp'

config device 'wan_dev'
        option name 'usb0'
        option macaddr '3c:46:d8:aa:89:50'

config interface 'wan6'
        option ifname 'usb0'
        option proto 'dhcpv6'

config switch
        option name 'switch0'
        option reset '1'
        option enable_vlan '1'

config switch_vlan
        option device 'switch0'
        option vlan '1'
        option ports '0 1 2 3 6t'

```

But i dont know how to set static IP on usb0 interface with ip 192.168.225.100, mask 255.255.255.0 gateway 192.168.225.1 and dns 192.168.225.1